account-keys/Cargo.toml

```
@@ -8,7 +8,7 @@ readme = "README.md"
 [dependencies]
 # External dependencies
 displaydoc = { version = "0.2", default-features = false }
-hkdf = { version = "0.9.0", default-features = false }
+hkdf = { version = "0.12.1-pre.0", default-features = false }
```

HKDF has no default features, so we can simplify this.
```suggestion
hkdf = "0.12.1-pre.0"
```